changelogs/fragments/479_enable_auto_commit.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - mysql_user - enable auto_commit to avoid MySQL metadata table lock (https://github.com/ansible-collections/community.mysql/pull/483)
```

```suggestion
  - mysql_user - enable auto_commit to avoid MySQL metadata table lock (https://github.com/ansible-collections/community.mysql/issues/479).
```
for consistency with other changelog entries (we put a link to a PR itself when there's no issue)

---

@GhostLyrics could you please merge this suggestion ^ ? after that we'll merge the PR